lib/matplotlib/rcsetup.py

```diff
@@ -60,8 +62,28 @@ def __call__(self, s):
                          % (self.key, s, list(six.itervalues(self.valid))))
 
 
+def _listify_validator(scalar_validator):
+    def f(s):
+        if isinstance(s, six.string_types):
+            try:
+                return [scalar_validator(v.strip()) for v in s.split(',')
+                        if v.strip()]
+            except Exception:
+                # Sometimes, a list of colors might be a single string
+                # of single-letter colornames. So give that a shot.
+                return [scalar_validator(v.strip()) for v in s if v.strip()]
+        elif type(s) in (list, tuple):
+            return [scalar_validator(v) for v in s if v]
+        else:
+            msg = "'s' must be of type [ string | list | tuple ]"
+            raise ValueError(msg)
+    f.__doc__ = scalar_validator.__doc__
+    return f
+
```

Comment:
Ahh yes, I thought we had properties that could take ``None``, but iirc in all situations that just means use the value in the rcParam...

Hmm, I didn't realise the cycler did that, I don't think we need that here as we deprecate this rcParam in favour of the ``prop_cycle``.  However if we do allow validate_colorlist to accept a splitable string, then I would prefer we did it there and not in a generic function...

Another thought this only applies for properties getting passed to a cycler, as you mentioned before, these validators can get used elsewhere. not just for the rcParams, so we shouldn't assume that this validator just gets used with the cycler.